README.md

```diff
@@ -1,146 +1,164 @@
-
-**Note 2:** If you're not using a cloud provider you need to make sure that you can [load balance](https://kubernetes.github.io/ingress-nginx/deploy/baremetal/) and [expose  applications](https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/#exposing-the-service) and provide [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
-
-You need to have following tools installed in your machine to install the stack:
-* Git
-* Java (Used by `keystore-init` script to generate keys)
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* [helm](https://github.com/helm/helm#install) (Server side component of Helm should be installed in Kubernetes cluster as well)
-* [helm-diff](https://github.com/databus23/helm-diff#install)
-* [helmfile](https://github.com/roboll/helmfile#installation)
-
-After installing them run following commands:
-```shell
-git clone --recurse-submodules https://github.com/RADAR-base/RADAR-Kubernetes.git
-cd RADAR-Kubernetes
-cp environments.yaml.tmpl environments.yaml
-cp base.yaml production.yaml
-cp base.yaml.gotmpl production.yaml.gotmpl
-vim environments.yaml # use the files you just created
-vim production.yaml  # Change setup parameters and configurations
-vim production.yaml.gotmpl  # Change setup parameters that require Go templating, such as reading input files
-./bin/keystore-init
-helmfile sync --concurrency 1
-```
+RADAR-Kubernetes setup uses [Helm](https://github.com/helm/helm) charts to package necessary Kubernetes resources for each component and [helmfile](https://github.com/roboll/helmfile) to modularize and deploy Helm charts of the platform on a Kubernetes cluster. This setup is designed to be a lightweight way to install and configure the RADAR-base components. The original images or charts may provide more and granular configurations. Please visit the `README` of respective charts to understand the configurations and visit the main repository for in depth knowledge.
 
-Having `--concurrency 1` will make installation slower but it is necessary because some components such as `kube-prometheus-stack` and `kafka-init` (aka `catalog-server`) should be installed in their specified order, if you've forgotten to use this flag the installation will not be successful and you need to go some cleaning before you can try to install again:
+## Status
+RADAR-Kubernetes is one of the youngest project of RADAR-base and will be the **long term supported form of deploying the platform**. Even though, RADAR-Kubernetes is being used in few production environments, it is still in its early stage of development. We are working improving the set up and documentation to enable RADAR-base community to make use of the platform.
```

Comment:
To Perhaps note to feedback questions/issues to this repo or the repo of the respective helm chart (if we are using submodules)?

Reply:
actually, I see something along these lines in the last section :+1: